### PR TITLE
Fixed invalid PresentationML parsing of recently added elements

### PR DESCRIPTION
**Pull request by symphony-thibault**
`MessageML` is a superset of `PresentationML`. It means that any `PresentationML` message should be parseable as valid `MessageML`. 

We noticed that we've missed this rule when implementing recent `MessageML` capabilities such as [`<ui-action>`](https://docs.developers.symphony.com/building-bots-on-symphony/messages/overview-of-messageml/ui-action-for-your-bots-ui-extensibility), [`<dialog>`](https://docs.developers.symphony.com/building-bots-on-symphony/messages/overview-of-messageml/ui-action-for-your-bots-ui-extensibility/dialog) or [`<time-picker>`](https://docs.developers.symphony.com/building-bots-on-symphony/messages/overview-of-messageml/symphony-elements-1/time-picker) elements. 

Most of the examples have been taken from our [Developer Documentation](https://docs.developers.symphony.com/building-bots-on-symphony/messages/overview-of-messageml) or our [PresentationML Live Renderer Tool](https://renderer-tool.app.symphony.com/).

### CODE REVIEW

File: src/test/java/org/symphonyoss/symphony/messageml/Mml2Pml2Pml.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+import javax.xml.parsers.DocumentBuilderFactory;
+
+/**
+ * This test loads a set of MessageML files from classpath:/examples location and performs 2 successive transformation:
+ * <ul>
+ *   <li>MessageML -> PresentationML</li>
+ *   <li>PresentationML -> PresentationML</li>
+ * </ul>
+ * It allows to ensure that generated PresentationML is an actual MessageML valid one.
```
Comment (symphony-youri) [inline src/test/java/org/symphonyoss/symphony/messageml/Mml2Pml2Pml.java:42]: should we check the markdown conversion too?

File: src/test/java/org/symphonyoss/symphony/messageml/Mml2Pml2Pml.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    log.info("1 # MML -> PML:");
+    MessageMLContext context = parse(message, entityJson);
+    log.info("\n{}\n{}{}", LINE, prettyXml(context.getPresentationML()), LINE);
+    //
+    // STEP 2 / PML -> PML
+    //
+    log.info("2 # PML -> PML:");
+    context = parse(context.getPresentationML(), context.getEntityJson());
+    context.getText();
+    log.info("\n{}\n{}{}", LINE, prettyXml(context.getPresentationML()), LINE);
```
Comment (symphony-youri) [inline src/test/java/org/symphonyoss/symphony/messageml/Mml2Pml2Pml.java:67]: should we actually (or did I miss it?) check that presentation ml is actually the same (is it or can it change a bit)?

Comment (symphony-thibault) [reply src/test/java/org/symphonyoss/symphony/messageml/Mml2Pml2Pml.java:67]: Actually for some elements, we introduce a random `id` prefix when converting MML -> PML ([here](https://github.com/finos/messageml-utils/blob/master/src/main/java/org/symphonyoss/symphony/messageml/elements/Dialog.java#L105) for instance) which prevent us from comparing the 2 PresentationMLs

Comment (symphony-thibault) [reply src/test/java/org/symphonyoss/symphony/messageml/Mml2Pml2Pml.java]: Done [here](https://github.com/finos/messageml-utils/pull/315/files#diff-878b6ef1d28806245245308ed46532eda41aa60eb7c43f9656cda16938430f47R77-R79) @symphony-youri, I used [xmlunit](https://github.com/xmlunit/xmlunit) to exclude properties with random values from XML tree comparison

File: src/test/java/org/symphonyoss/symphony/messageml/Mml2Pml2Pml.java (hunk 1 above)
Comment (symphony-thibault) [reply src/test/java/org/symphonyoss/symphony/messageml/Mml2Pml2Pml.java:42]: Yes, done [here](https://github.com/finos/messageml-utils/pull/315/files#diff-878b6ef1d28806245245308ed46532eda41aa60eb7c43f9656cda16938430f47R74)

**Review by symphony-mariacristina — APPROVED** [review]
LGTM!

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**